azure-storage-cpp-lite/include/retry.h

```diff
@@ -67,8 +68,11 @@ namespace microsoft_azure {
             retry_info evaluate(const retry_context &context) const override {
                 if (context.numbers() == 0) {
                     return retry_info(true, std::chrono::seconds(0));
-                } else if (context.numbers() < 100 && can_retry(context.result())) {
-                    return retry_info(true, std::chrono::seconds(5+rand()%5));
+                } else if (context.numbers() < 60 && can_retry(context.result())) {
+                    double delay = (pow(2, context.numbers()-1)-1);
+                    delay *= (((double)rand())/RAND_MAX)/2 + 0.8;
+                    delay = std::min(delay, 60.0); // Maximum backoff delay of 1 minute
```

Comment:
Can you move the jitter after the min(delay, 60) check?  This also doesn't matter too much, but I'd feel a bit more comfortable if the max-retry case has the jitter as well